### PR TITLE
Ran/new login

### PR DESCRIPTION
**Pull request by stormcrow85**

### CODE REVIEW

File: demo/loginForm/index.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
     <script src="infra/dist/svgIcon/svgIcon.js"></script>
 
     <script src="loginForm/loginForm.js"></script>
-    <script src="loginForm/templates.js"></script>
+    <!--<script src="loginForm/templates.js"></script>-->
```
Comment (assafshp) [inline demo/loginForm/index.html]: ?

File: demo/regComponent/demo.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    .run(function ($rootScope, $translate) {
+        $rootScope.$on('$translatePartialLoaderStructureChanged', function () {
+            $translate.refresh();
+        })
+    })
+    .service('ENV', function () {
+        this.fbGlobalEndPoint = "https://znk-dev.firebaseio.com/";
+        this.backendEndpoint = "https://znk-web-backend-dev.azurewebsites.net/";
+        this.fbDataEndPoint = "https://act-dev.firebaseio.com/";
+        this.dataAuthSecret = "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VyIjoicmFjY29vbnMifQ.mqdcwRt0W5v5QqfzVUBfUcQarD0IojEFNisP-SNIFLM";
```
Comment (assafshp) [inline demo/regComponent/demo.js]: who needs this key?

File: demo/loginForm/index.html (hunk 1 above)
Comment (stormcrow85) [inline demo/loginForm/index.html]: templates.js returns not found, no such file

File: demo/regComponent/demo.js (hunk 2 above)
Comment (stormcrow85) [inline demo/regComponent/demo.js]: when logging in it is sent to the server, in this object:
```
                    var postData = {
                        email: authData.password ? authData.password.email : '',
                        uid: authData.uid,
                        fbDataEndPoint: ENV.fbDataEndPoint,
                        fbEndpoint: ENV.fbGlobalEndPoint,
                        auth: ENV.dataAuthSecret,
                        token: authData.token
                    };

```

**Pull request merged**